security/crowdsec/src/opnsense/service/conf/actions.d/actions_crowdsec.conf

```diff
@@ -0,0 +1,99 @@
+
+# https://docs.opnsense.org/development/backend/configd.html
+
+[start]
+command:/usr/local/etc/rc.d/oscrowdsec start
+type: script
+message: starting crowdsec services
+
+[stop]
+command:/usr/local/etc/rc.d/oscrowdsec stop
+type: script
+message: stopping crowdsec services
+
+[status]
+command:/usr/local/etc/rc.d/oscrowdsec status; exit 0
+type: script_output
+message: oscrowdsec status
+
+[restart]
+command:/usr/local/etc/rc.d/oscrowdsec restart
+type: script
+message: stopping crowdsec services
+
+[reload]
+command:/usr/local/etc/rc.d/oscrowdsec reload
+type: script
+message: reload crowdsec configuration
+
+[crowdsec-status]
+command:/usr/local/etc/rc.d/crowdsec status;exit 0
+type:script_output
+message: request crowdsec status
+
+[crowdsec-firewall-status]
+command:/usr/local/etc/rc.d/crowdsec_firewall status;exit 0
+type:script_output
+message: request crowdsec_firewall status
+
+[alerts-list]
+command:/usr/local/bin/cscli alerts list -l 0 -o json | sed 's/^null$/\[\]/'
+type:script_output
+message:crowdsec alerts list
+
+[bouncers-list]
+command:/usr/local/bin/cscli bouncers list -o json | sed 's/^null$/\[\]/'
+type:script_output
+message:crowdsec bouncers list
+
+[collections-list]
+command:/usr/local/bin/cscli collections list -o json
+type:script_output
+message:crowdsec collections list
+
+[decisions-list]
+command:/usr/local/bin/cscli decisions list -l 0 -o json | sed 's/^null$/\[\]/'
+type:script_output
+message:crowdsec decisions list
+
+[decisions-delete]
+command:/usr/local/bin/cscli --error decisions delete 2>&1
+parameters:--id %s
+type:script_output
+message:crowdsec decisions delete
+
+[machines-list]
+command:/usr/local/bin/cscli machines list -o json | sed 's/^null$/\[\]/'
+type:script_output
+message:crowdsec machines list
+
+[parsers-list]
+command:/usr/local/bin/cscli parsers list -o json
+type:script_output
+message:crowdsec parsers list
+
+[postoverflows-list]
+command:/usr/local/bin/cscli postoverflows list -o json
+type:script_output
+message:crowdsec postoverflows list
+
+[scenarios-list]
+command:/usr/local/bin/cscli scenarios list -o json
+type:script_output
+message:crowdsec scenarios list
+
+[version]
+command:/usr/local/bin/cscli version 2>&1
+type:script_output
+message:crowdsec version
+
+[reconfigure]
+command:/usr/local/opnsense/scripts/OPNsense/CrowdSec/reconfigure.sh
```

Comment:
reconfigure is nesting configd commands.. it should work but points to overly complex reconfiguration flow

Reply:
Yes it's complex because I have two required daemons (the bouncer won't restart if the agent is missing, etc..). The current flow seems stable but I can certainly simplify it in a next version.

Reply:
yes please :)